dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.cpp

```diff
@@ -23,12 +26,14 @@ namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
 
     winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationBuilder AppNotificationBuilder::SetTimeStamp(winrt::Windows::Foundation::DateTime const& value)
     {
-        
-        //winrt::std::wstring currentLanguage { ApplicationLanguages::Languages().First().Current() };
-        //// DateTimeFormatter dateTimeFormatter{ "longtime" };
-        //winrt::Windows::Foundation::DateTime::
-        //winrt::Windows::Foundation::DateTime { 2022, 1, 13, 16, 25, 30,  }
-        //m_timeStamp = timeStamp;
+        auto seconds{ winrt::clock::to_time_t(value) };
+        struct tm buf;
```

Comment:
NIT: `struct tm buf{};`

same comment applies throught

Reply:
Addressed in https://github.com/microsoft/WindowsAppSDK/pull/2801